.snippets/code/tutorials/polkadot-sdk/parachains/system-chains/asset-hub/send-tx-paying-fees-with-different-tokens/fee-payment-transaction.ts

```diff
@@ -0,0 +1,69 @@
+import { sr25519CreateDerive } from "@polkadot-labs/hdkd";
```

Comment:
This file wasn't formatted properly. Makes me sad that the pre-commit hook isn't working anymore. I'll create a ticket for one of the regulators to look into this and fix it.